src/gui/tray/UserModel.cpp

```diff
@@ -732,9 +733,10 @@ Q_INVOKABLE void UserModel::openCurrentAccountServer()
 
     QString url = _users[_currentUserId]->server(false);
     if (!(url.contains("http://") || url.contains("https://"))) {
+        // TODO: Maybe we need to fix the condition? It looks totally wrong. Should probably use QString::startsWith and check for !url.startsWith("https://") ?
         url = "https://" + _users[_currentUserId]->server(false);
     }
-    QDesktopServices::openUrl(QUrl(url));
+    Utility::openBrowser(QUrl(url), nullptr);
```

Comment:
Not needed for that one because of the check in the block just before. You are right that this use of "contains" is surprising and "startsWith" would be more appropriate.